scale/recipe/messages/purge_recipe.py

```diff
@@ -7,28 +7,32 @@
 
 from batch.models import BatchRecipe
 from job.messages.spawn_delete_files_job import create_spawn_delete_files_job
+from messaging.messages.message import CommandMessage
 from recipe.definition.node import JobNodeDefinition, RecipeNodeDefinition
 from recipe.models import Recipe, RecipeInputFile, RecipeNode
-from messaging.messages.message import CommandMessage
+from source.messages.purge_source_file import create_purge_source_file_message
 
 
 logger = logging.getLogger(__name__)
 
 
-def create_purge_recipe_message(recipe_id, trigger_id):
+def create_purge_recipe_message(recipe_id, trigger_id, purge):
     """Creates messages to remove the given recipe by ID
 
     :param recipe_id: The recipe ID
     :type purge_job_ids: int
     :param trigger_id: The trigger event ID for the purge operation
     :type trigger_id: int
+    :param purge: Boolean value to determine if files should be purged from workspace
+    :type purge: bool
     :return: The purge recipe message
     :rtype: :class:`recipe.messages.purge_recipe.PurgeRecipe`
     """
 
     message = PurgeRecipe()
     message.recipe_id = recipe_id
     message.trigger_id = trigger_id
+    message.purge = purge
```

Comment:
Do you need a purge flag for this message? Since it is purging, you know purge will always be true.